docs/guides/admin/docs/installation/rpm-el7.md

```diff
@@ -0,0 +1,190 @@
+Install from Repository (Red Hat Enterprise Linux 7.x, CentOS 7.x, Scientific Linux 7.x)
+========================================================================================
+
+> *This guide is for EL7 only. There is a separate [CentOS 8 and Red Hat Enterprise Linux 8 guide](rpm-el8.md).*
+
+This guide is based on a RPM software repository available for Red Hat-based Linux distributions provided by Osnabrück
+University. This repository provides preconfigured Opencast installations and all necessary 3rd-Party-Tools. Using this
+method, you do not have to compile the software by yourself.
+
+
+Availability
+------------
+
+Note that it may take some time (usually about two weeks after a new release is out) before the RPMs are available.
+Watch for announcements on [the users list](https://docs.opencast.org/#mailing-lists) or just check which versions are
+available in the repository.
+
+
+Currently Supported
+-------------------
+
+- CentOS 7.x (x86\_64)
+- Red Hat Enterprise Linux 7.x (x86\_64)
+- Scientific Linux 7.x (x86\_64)
+
+> *Other architectures like i386, i686, arm, … are not supported!*
+
+
+Registration
+------------
+
+Before you can start you need to get an account for the repository. You will need the credentials that you got via
+e-mail after the registration to successfully complete this guide.
+
+Get your account on [pkg.opencast.org](https://pkg.opencast.org).
+
+
+Activate Repository
+-------------------
+
+First you have to install the necessary repositories so that your package manager can access them:
+
+1. Add Opencast repository:
+
+        cd /etc/yum.repos.d
+        curl -O https://pkg.opencast.org/opencast.repo \
+           -d os=el -d version=7 -u [YOUR_USERNAME]
+
+    You will be asked for your password.
+
+    It might take some time after the release of a new Opencast version before the RPMs are moved to the stable
+    repository. Until then, you can use `…/opencast-testing.repo` instead to get the latest version. Note that the
```

Comment:
I did intentionally avoid that so that people don't just copy and paste that command without thinking. If someone enables this, that should be intentional.